### PR TITLE
Make polestar look like voyager

### PR DESCRIPTION
**Pull request by vlandham**
Here is a first pass at making polestar look like voyager.

I took an unguided stab at a new logo. If there is some traction there - we can iterate on it - or we can just drop it. 

There is a VLUI PR here: https://github.com/vega/vega-lite-ui/pull/119

screen shot 2015-10-16 at 11 58 04 am

logo:

logo

### CODE REVIEW

**Comment by vlandham** [comment]
ping @kanitw @domoritz 

if we like this direction, I'll add a PR for voyager to remove the scss from there. thanks!

**Comment by vlandham** [comment]
and again - the logo was just a quick experiment. in no way am i emotionally attached to it ;)

**Comment by kanitw** [comment]
1) I think this is definitely a direction to go.  Nice start! :)  

2) As mentioned in https://github.com/vega/voyager/issues/206, I think the encoding tab should have a lighter dark grey background (lighter than the data). 

2.1) With that background, we can remove the grey border between cells and make the header background slightly darker?  (Same should be applied to Voyager -- see https://github.com/vega/voyager/issues/211)
Maybe I'll take stab at this in Voyager first so feel free to ignore this (2.1) for now.  

3) For the logo, I think we should definitely use consistent typeface and also font size.  I didn't bring my main laptop, which has the sketch file for the latest voyager logo, so I can't send you now.  Maybe you can send me the polestar logo sketch file?  I can help adjust font-size to be consistent.  

The icon looks really nice for a first stab and we could iterate on that, or try a few more ideas if we have time.   Maybe we can rotate the stars a bit make make the solid star straight and only rotate the lighter star?

**Comment by vlandham** [comment]
for the logo - its the same typeface - but the font might look a bit differently sized. I'll send you the sketch file

**Comment by kanitw** [comment]
Thanks.  (FYI, I meant I like that you use the same typeface, sorry for not being clear on that :) )

**Comment by domoritz** [comment]
I love the logo!

**Comment by vlandham** [comment]
attempting to add color to the shelf. it starts to look a bit heavy on that side...

thoughts?
screen shot 2015-10-19 at 1 44 46 pm

**Comment by kanitw** [comment]
Yeah that's quite heavy, sorry for bad suggestions.  Another idea to try add light grey that I end up using for voyager? (`#F1F4F9`)

Note that in Voyager, I ended up making section header __lighter__ than the background, but still darker than white.  (See screenshot in https://github.com/vega/voyager/pull/217)

**Comment by kanitw** [comment]
While https://github.com/vega/vega-lite-ui/pull/131 is pending, you might want to check out that branch first so you use the latest VLUI.

File: src/app/main/main.html [hunk 1]
```diff
@@ -1,43 +1,49 @@
...
-    <div class="card no-right-margin no-top-margin">
-      <div class="right flex-root hflex">
-        <div class="controls">
-          <span ng-show="consts.debug">
-            <a ng-href="{{ {encoding:Spec.chart.vlSpec} | reportUrl }}" target="_blank" class="command debug" tooltips tooltip-size="small"
-              tooltip-content="Report an issue">
-              <i class="fa fa-bug"></i>
+    <div class="card top-card no-right-margin no-top-margin">
+      <div class="hflex">
+        <img id="logo" src="../assets/images/polestar-logo.png"></img>
```
Comment (domoritz) [inline src/app/main/main.html]: Please use div like I did in voyager. Otherwise we cannot deploy.

Comment (kanitw) [inline src/app/main/main.html]: I don’t know what you mean…

If it’s easier, please go ahead and fix it.

**Comment by domoritz** [comment]
I feel like the IDL logo has gotten worse (here and in voyager). Can we undo that change?

**Comment by kanitw** [comment]
@domoritz  we should use a bigger one and resize to this size, I think the quality of image is worse but the displayed size is better.    The original logo size was too big and does __not__ respect header bar's padding/margin.  That said, it can be a little bigger than the current one.

**Comment by kanitw** [comment]
I’ll resize a bit now..

**Pull request closed without merging**